e2e-tests/cypress/tests/integration/channels/messaging/emoji_picker_keyboard_usability_spec.js

```diff
@@ -37,7 +37,7 @@ describe('MM-13064 - Emoji picker keyboard usability', () => {
             and('have.focus').
             type('si').
             should('have.value', 'si').
-            type('{leftarrow}m').
+            type('{leftarrow}{leftarrow}m').
```

Comment:
Do we have to press twice ?

Reply:
@M-ZubairAhmed on 9.6 version this is not needed.
On latest master, pressing left arrow twice gets the focus from emoji to search box and move the cursor.
{leftarrow} is to bring the focus back

Reply:
Is it as expected?

Reply:
@M-ZubairAhmed , can you confirm ?